tests/monotouch-test/Security/SecStatusCodeTest.cs

```diff
@@ -32,9 +33,14 @@ public void ErrorDescriptionTest ()
 			var desc = SecStatusCode.Success.GetStatusDescription ();
 			Assert.NotNull (desc, $"{nameof (desc)} not null");
 
-			// This value generated from objc enum documentation and very unlikely to change.
-			var noErr = "No error.";
-			Assert.That (desc, Is.EqualTo (noErr), $"{nameof (desc)} == {noErr}");
+			var lang = CultureInfo.CurrentUICulture.TwoLetterISOLanguageName;
+			if (lang.Equals ("en", StringComparison.InvariantCultureIgnoreCase)) {
```

Comment:
you compare native (iOS) and .net languages here
we try our best but they can mismatch (e.g. iOS allows "DE-fr" while .net does not)
would be safer to use native API to retrieve the current language